.changeset/chilled-geese-sniff.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ux/axios-extension': minor
```

Comment:
patch should be fine since your fixing a bug

Reply:
Updated changeset, thanks @tobiasqueck